hnn_core/cell_response.py

```diff
@@ -75,13 +75,17 @@ class CellResponse(object):
     """
 
     def __init__(self, spike_times=None, spike_gids=None, spike_types=None,
-                 times=None, cell_type_names=None):
+                 times=None, cell_type_names=None, vsec=None, isec=None):
```

Comment:
write method and `read_cell_response` to be updated as well

Reply:
you need to document `vsec` and `isec` ?

Reply:
this falls under the network saving PR now so no longer needed to update here